code/game/gamemodes/objective.dm

```diff
@@ -768,5 +768,5 @@ var/list/potential_theft_objectives = subtypesof(/datum/theft_objective) \
 //wizard
 
 /datum/objective/wizchaos
-	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless Nanotrasen scum a message!"
+	explanation_text = "Wreak havoc upon the station as much you can. Send those wandless NanoTrasen scum a message!"

```

Comment:
A very long time ago: https://github.com/ParadiseSS13/Paradise/pull/3

Any instance of `NanoTrasen` is in error and should be corrected.